zarr_etl_tools/utils/zarr_methods.py

```diff
@@ -515,16 +549,8 @@ def write_initial_zarr(self):
         mapper = self.store.mapper(set_root=False)
 
         self.to_zarr(dataset, mapper, consolidated=True, mode="w")
-
         if isinstance(self.store, IPLD):
             self.dataset_hash = str(mapper.freeze())
-            self.info(f"IPFS hash is {str(self.dataset_hash)}")
-
-            # Create and publish metadata as a STAC Item
-            self.create_root_stac_catalog()
-            self.create_stac_collection(dataset)
-            self.create_stac_item(dataset, self.dataset_hash)
-            self.info("Published dataset's IPFS hash is " + str(self.dataset_hash))
```

Comment:
please keep a line printing the new hash to INFO somewhere in the standard IPFS parse, it’s very helpful in a multitude of situations

Reply:
I moved this to `parse`, where I think it makes more sense